tools/travis-test.sh

```diff
@@ -167,7 +167,12 @@ run_tests() {
   echo
   echo "All tests done."
 
-  if [ $SMALL_STATUS -eq 0 -a $BIG_STATUS -eq 0 -a $BIG_STATUS_BY_SUMMARY -eq 0 ]
+  grep "fail_ci_build=true" ${BASE}/dev/mongooseim_*/log/ejabberd.log
```

Comment:
The path to logs is wrong. It should be `_build/mim*/rel/mongooseim/log/ejabberd.log`